indra_network_search/data_models/rest_models.py

```diff
@@ -0,0 +1,33 @@
+"""
+Contains return models from the rest api
+"""
+from pydantic import BaseModel
+from typing import Optional
+from datetime import date
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
+
+__all__ = ["Health", "ServerStatus"]
+
+
+STR_STATUS = Literal["booting", "available"]
+
+
+class Health(BaseModel):
+    """Health status"""
+
+    status: STR_STATUS
+
+
+class ServerStatus(BaseModel):
+    """Status with more detail than health"""
+
+    unsigned_nodes: Optional[int] = None
```

Comment:
It would be nice to add the pydantic docs for these fields
```suggestion
        unsigned_nodes: Optional[int] = field(..., description="...")
```

Reply:
Cool, I didn't know that was a thing